src/OpenTelemetry.ClrProfiler.Managed/Configuration/EnvironmentConfigurationHelper.cs

```diff
@@ -19,13 +19,7 @@ internal static class EnvironmentConfigurationHelper
 
         public static TracerProviderBuilder UseEnvironmentVariables(this TracerProviderBuilder builder, Settings settings)
         {
-            string serviceName = settings.ServiceName
-                ?? GetApplicationName()
-                ?? "UNKNOWN_SERVICE_NAME";
-
-            var resourceBuilder = ResourceBuilder
-                .CreateDefault()
-                .AddService(serviceName, serviceVersion: settings.ServiceVersion);
+            var resourceBuilder = ResourceBuilder.CreateDefault();
```

Comment:
the .NET OTel SDK has different default behavior when the service name is not assigned. it sets `unknown_service:{processName}`

Reply:
Ideally the SDK should try the name of the entry assembly first and only if that fails fallback to the process name. This way it gets better default names for applications started via `dotnet MySvc.dll` which will be reporting `dotnet` as the process name.

If one of us put a PR fixing that on the SDK it will likely be accepted.

@cijothomas let's know if you see any problem with the change that I'm suggesting above for the default service name.